bee-node/bee-plugin/bee-plugin-version-checker/src/release_info.rs

```diff
@@ -0,0 +1,59 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use semver::Version;
+use serde::Deserialize;
+
+use std::cmp;
+
+#[derive(Deserialize, Debug, Clone)]
+pub struct ReleaseInfoBuilder {
+    pub html_url: String,
+    pub tag_name: String,
+}
+
+impl ReleaseInfoBuilder {
+    /// Attempts to build a [`ReleaseInfo`].
+    ///
+    /// Returns:
+    ///  - `None` if there is an error parsing a version from the release tag.
+    ///  - `None` if this is a pre-release, and the current `bee-node` is *not* a pre-release.
+    ///  - `Some` otherwise.
+    pub(crate) fn build(self, pre_release: bool) -> Option<ReleaseInfo> {
+        let mut version = self.tag_name.clone();
+        version.retain(|c| c != 'v');
```

Comment:
I know this should be fine in pretty much all cases but it still seems a bit weird in case for some reason there is another `v`. Maybe we can be a bit smarter and only remove v if it's the first char of the tag ?